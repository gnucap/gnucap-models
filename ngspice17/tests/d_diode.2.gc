attach mgsim
* simple diode test
spice
V1   1  0  10.
R1   1  2  50.K
D1   2  0  ddd  area=1.
.model  ddd  d  ( is= 10.f  rs= 10k  n= 1.  tt= 0.  cjo= 0.  vj= 1.  m= 0.5 
+ eg= 1.11  xti= 3.  kf= 1.  af= 1.  fc= 0.5  )
.verilog

print op v(1 2) i(V1)
op
end
