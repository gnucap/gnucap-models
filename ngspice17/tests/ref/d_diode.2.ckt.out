simple diode test 
#           v(1)       v(2)      
 27.        10.        2.1726    
